test/examples/to-fix/simplified-validty-predicate/example1.ju

```diff
@@ -0,0 +1,28 @@
+mod Example where
+
+open Prelude
+open Michelson
+
+-- What we want supported
+
+-- Basic record declaration
+type account = {
+  balance : nat
+}
+
+-- Even records with 2 elements in them!
+type transaction-type = {
+  account-from : account,
+  account-to   : account
+}
+
+type storage = {
+  transaction : transaction-type,
+}
+
+sig accept-withdraws-from : key -> storage -> storage -> bool
+let accept-withdraws-from key {transaction = initial} {transaction = final}
```

Comment:
What is this `{transaction = initial}` syntax? Is that already supported? If not, it should be added as a comment and maybe discuss its priority / plan of action